cmd/atlas/inspect.go

```diff
@@ -12,16 +16,17 @@ var (
 	inspectCmd = &cobra.Command{
 		Use:   "inspect",
 		Short: "Inspect atlas schema.",
-		Run:   func(cmd *cobra.Command, args []string) {},
+		Run:   func(cmd *cobra.Command, args []string) { runInspect(inspectFlags.dsn) },
 		Example: `
-atlas schema inspect -d mysql://user:pass@host:port/dbname
-atlas schema inspect -d postgres://user:pass@host:port/dbname
-atlas schema inspect --dsn sqlite3://path/to/dbname.sqlite3`,
+atlas schema inspect -d "mysql://user:pass@host:port/dbname"
+atlas schema inspect -d "postgres://user:pass@host:port/dbname"
+atlas schema inspect --dsn "sqlite3://path/to/dbname.sqlite3"`,
```

Comment:
why is this necessary?

Reply:
zsh shell has problems with url that is domain without string.

Reply:
https://github.com/ohmyzsh/ohmyzsh/issues/31

Reply:
I work with zsh and it works for me. 

```go
// go run main.go --dsn sqlite3://path/to/dbname.sqlite3
// prints: [.../exe/main --dsn sqlite3://path/to/dbname.sqlite3]
func main() {
	fmt.Println(os.Args)
}
```

Reply:
btw, remove sqlite from the examples as it's not supported atm